devices/flash-flexspi/fspi/fspi_rt105x.h

```diff
@@ -30,9 +30,9 @@ enum { mcr0 = 0, mcr1, mcr2, ahbcr, inten, intr, lutkey, lutcr, ahbrxbuf0cr0, ah
 #define AHBRXBUF_CNT 4
 
 
-static inline void *flexspi_ahbAddr(int instance)
+static inline addr_t flexspi_ahbAddr(int instance)
 {
-	return instance == flexspi_instance1 ? (void *)0x60000000 : NULL;
+	return instance == flexspi_instance1 ? 0x60000000 : 0;
```

Comment:
MISRA
```c
return (instance == flexspi_instance1) ? 0x60000000 : 0;
```

Reply:
I'll post a separate PR about `flexspi` that enforces MISRA style and rules, many changes are required, let this commit change be atomic. Thanks.